3-probability/03-random-variables/notes.qmd

```diff
@@ -180,7 +180,7 @@ Let $X$ be the number of individuals in our sample who have had at least one pos
 Then $X \sim Bin(10, p = 0.35)$. (Why are these the parameters?)
 Using the complement rule, 
 
-$P(X > 3) = 1 - P(X \le 3) = 1 - \left(P(X = 0) + P(X = 1) + P(X = 2)\right)$. 
+$P(X > 3) = 1 - P(X \le 3) = 1 - \left(P(X = 0) + P(X = 1) + P(X = 2) + P(X = 2)\right)$.
```

Comment:
This should be P(X = 3) right?

Reply:
yes, oops. I think Iain tried to fix it :). I will fix it now.